### DISCUSSION TITLE
Virtuellmenu

### DISCUSSION THREAD

**Discussion by leanderlingens** [Zrips/CMI#3870] [Feature Requests]
The command /anvil and the command /workbench for:

- furnace | /furnace
- brewing_stand | /brewingstand
- enchanting_table | /enchantingtable
- smoker | /smoker
- blast_furnace | /blastfurnace
- cartography_table | /cartographytable
- fletching_table | /fletchingtable
- grindstone | /grindstone
- smithing_table | /smithingtable
- stonecutter | /stonecutter

**Comment by RSLuke** [comment]
It would be nice to have these commands added indeed

**Comment by byjdw** [comment]
bump

**Comment by CloudeLecaw** [comment]
some allready exists my you didnt noticed?

```
cmi cartographytable (playerName)
Open cartography table
```

```
cmi grindstone (playerName)
Open grindstone
```

```
cmi workbench (playerName)
Open workbench
```

```
cmi anvil (playerName)
Open anvil
```

```
cmi smithingtable (playerName)
open smithingable
```

```
cmi stonecutter (playerName)
open stonecutter
```

the once that need a virtual buffer for workage process dont exsits (except the fletching table) so not eisting once are:

furnace | /furnace
brewing_stand | /brewingstand
enchanting_table | /enchantingtable
smoker | /smoker
blast_furnace | /blastfurnace
fletching_table | /fletchingtable
